landice/output_processing_li/plot_globalStats.py

```diff
@@ -97,6 +100,15 @@
 plt.grid()
 
 
+def VAF2seaLevel(vol):
+    return vol * scaleVol / 3.62e14 * rhoi / rhosw * 1000.
```

Comment:
What do you think about making this always relative to the initial sea level equivalent? i.e.
```suggestion
    return (vol - vol[0]) * scaleVol / 3.62e14 * rhoi / rhosw * 1000.
```
(and similar for the inverse)

Reply:
@matthewhoffman, this would only work if all the runs plotted are starting from the same VAF, which won't always be the case. So we can either have both axes show totals or change from initial, but we can't have the VAF axis show total and the SLR axis show change from initial if we want to maintain generality. I'd be fine with either way. Let me know which one you'd prefer.

Reply:
@matthewhoffman, are you happy with this, or would you like me to make both axes show change from the initial time?